modules/integration_azure_firewall/conf/02-throughput.yaml

```diff
@@ -0,0 +1,27 @@
+## Example
+module: firewall
+name: throughput
+
+transformation: false
+aggregation: true
+filtering: "filter('resource_type', 'Microsoft.Network/azureFirewalls') and filter('primary_aggregation_type', 'true')"
+
+tip: "Firewall will autoscale at 2.5Gbps but the maximum throughput is 30Gbps"
+
+signals:
+  signal:
+    metric: "Throughput"
+rules:
+  minor:
+    threshold: 335544320
+    comparator: ">"
+    lasting_duration: "900s"
+  critical:
+    threshold: 3355443200
+    comparator: ">"
+    dependency: minor
+    lasting_duration: "900s"
+  major:
+    threshold: 4026531840
+    comparator: ">"
+    dependency: critical
```

Comment:
it seems strange to use lasting for others rules but notthis one

Reply:
It's voluntary, This is a hard limit and I want an immediate alert if reached.